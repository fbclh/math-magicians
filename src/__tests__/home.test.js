import renderer from 'react-test-renderer';
import Home from '../components/Home';

it('Check if Home page renders correctly', () => {
  const tree = renderer.create(<Home />);
  expect(tree).toMatchSnapshot();
});
