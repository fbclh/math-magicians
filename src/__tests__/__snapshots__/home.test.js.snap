// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if Home page renders correctly 1`] = `
<div
  className="main-div"
>
  <h2
    className="welcome"
  >
    Welcome to our page
  </h2>
  <p
    className="text-p"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Augue ut lectus arcu bibendum. Neque sodales ut etiam sit. Risus in hendrerit gravida rutrum. Egestas quis ipsum suspendisse ultrices gravida dictum. Etiam dignissim diam quis enim lobortis. Aliquet nec ullamcorper sit amet risus nullam. Nec dui nunc mattis enim. Sit amet nisl purus in mollis nunc sed.
  </p>
</div>
`;
